Error while checking build/O2Physics/o2 for 710f4bbd9cea186b7c887b07ec5caed23b202b36 at 2023-04-11 15:22:
```
## sw/BUILD/O2Physics-latest/log
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:32: error: 'CompatibleColls' does not name a type; did you mean 'CompatibleCollIds'?
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: 'binding_t' does not name a type
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: 'binding_t' was not declared in this scope
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: no matching function for call to 'o2::aod::track_association::CompatibleCollIds::compatibleColl_as<<expression error> >() const'
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: template argument 1 is invalid
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: 'binding_t' was not declared in this scope
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: no matching function for call to 'o2::aod::track_association::CompatibleCollIds::compatibleColl_first_as<<expression error> >() const'
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: template argument 1 is invalid
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: 'binding_t' was not declared in this scope
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: no matching function for call to 'o2::aod::track_association::CompatibleCollIds::compatibleColl_last_as<<expression error> >() const'
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: template argument 1 is invalid
/sw/SOURCES/O2Physics/2366-slc7_x86-64/0/Common/DataModel/CollisionAssociation.h:27:1: error: 'binding_t' was not declared in this scope
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/2366/710f4bbd9cea186b7c887b07ec5caed23b202b36/build_O2Physics_o2/pretty.html).

--- a/Common/DataModel/CollisionAssociation.h
+++ b/Common/DataModel/CollisionAssociation.h
@@ -32,7 +32,7 @@ DECLARE_SOA_TABLE(TrackAssoc, "AOD", "TRACKASSOC", //! Table for track-to-collis
                   track_association::TrackId);
 
 DECLARE_SOA_TABLE(TrackCompColls, "AOD", "TRACKCOMPCOLLS", //! Table with vectors of collision indices stored per track
-                  track_association::CompatibleCollIds);
+                  track_association::CollisionIds);
 } // namespace o2::aod
 
 #endif // COMMON_DATAMODEL_COLLISIONASSOCIATION_H_

--- a/Common/DataModel/CollisionAssociation.h
+++ b/Common/DataModel/CollisionAssociation.h
@@ -24,7 +24,7 @@ namespace track_association
 {
 DECLARE_SOA_INDEX_COLUMN(Collision, collision);                      //! Collision index
 DECLARE_SOA_INDEX_COLUMN(Track, track);                              //! Track index
-DECLARE_SOA_ARRAY_INDEX_COLUMN(Collision, compatibleColl);      //! Array of collision indices
+DECLARE_SOA_ARRAY_INDEX_COLUMN(Collision, compatibleColl);           //! Array of collision indices
 } // namespace track_association
 
 DECLARE_SOA_TABLE(TrackAssoc, "AOD", "TRACKASSOC", //! Table for track-to-collision association for e.g. HF vertex finding - tracks can appear for several collisions

--- a/Common/DataModel/CollisionAssociation.h
+++ b/Common/DataModel/CollisionAssociation.h
@@ -24,7 +24,7 @@ namespace track_association
 {
 DECLARE_SOA_INDEX_COLUMN(Collision, collision);                      //! Collision index
 DECLARE_SOA_INDEX_COLUMN(Track, track);                              //! Track index
-DECLARE_SOA_ARRAY_INDEX_COLUMN(CompatibleColl, compatibleColl);      //! Array of collision indices
+DECLARE_SOA_ARRAY_INDEX_COLUMN(Collision, compatibleColl);      //! Array of collision indices
 } // namespace track_association
 
 DECLARE_SOA_TABLE(TrackAssoc, "AOD", "TRACKASSOC", //! Table for track-to-collision association for e.g. HF vertex finding - tracks can appear for several collisions

